src/BODSDataExtractor/extractor.py

```diff
@@ -571,12 +571,73 @@ def analytical_timetable_data_analysis(self):
         
         if self.stop_level == True:
             self.service_line_extract = self.service_line_extract_with_stop_level_json.drop(['journey_pattern_json','vehicle_journey_json','services_json','la_code'],axis=1).drop_duplicates()
+            self.check_for_expired_operators()
         else:
             self.service_line_extract = self.service_line_extract_with_stop_level_json.drop(['la_code'],axis=1).drop_duplicates()
-        
+            self.check_for_expired_operators()
         return self.service_line_extract
 
 
+
+    def expired_status(date, end_date, today):
+        """
+        Based on the expiry date, a boolean value is assigned to the 'expired' variable, this is
+        then returned back to the 'check_for_expired_operators' function.
+        """        
+        
+        expired=False
+        
+        #If the end date is greater or equal to today's date, if so expiry is False 
+        if end_date>=today:
+            expired=False
+                                       
+        #If the end date is less than today's date, if so expiry is True                               
+        elif end_date<today:
+            expired=True
+                                       
+        else:
+            print("Unknown")

```

Comment:
need a more descriptive comment to the user than 'unknown' - please outline what has actually happened so users can troubleshoot and report